example/listenvironments/main.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/google/go-github/v42/github"
+	"golang.org/x/oauth2"
+)
+
+// An example of how to use ListEnvironments method with EnvironmentListOptions.
+// It's runnable with the following command:
+// export GITHUB_TOKEN=your_token export GITHUB_REPOSITORY_OWNER=your_owner export GITHUB_REPOSITORY_NAME=your_repo && go run .
+func main() {
+
+	var client *github.Client
+	var ctx = context.Background()
```

Comment:
This line can be moved lower (around 35-36), closer to where it is actually being used.

Reply:
Moved as suggested. I've also swapped the var declaration for the short declaration here too.